examples/oauth-pat-generation/app.js

```diff
@@ -0,0 +1,93 @@
+const CLIENT_ID = "A664Jlyp4mpBSVIktWtGjeshNaERA_CBzSC_IcNl618";
```

Comment:
should this be an env var?

Reply:
I've updated this to remove my CLIENT_ID. This doesn't need to be an env var in this scenario, but definitely shouldn't have my ID there 😅